provisioning/service/src/Contract/ContractApiResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Net;
+
+namespace Microsoft.Azure.Devices.Provisioning.Service
+{
+    public class ContractApiResponse
+    {
+        public HttpStatusCode StatusCode { get; internal set; }
```

Comment:
Let's not use internal set with no validation.
Any reason not to have a ctor and make this immutable?

Reply:
Ctor added.

Reply:
Done